libraries/AP_HAL_PX4/RCOutput.cpp

```diff
@@ -399,34 +395,12 @@ void PX4RCOutput::read_last_sent(uint16_t* period_us, uint8_t len)
     }
 }
 
-/*
-  update actuator armed state
- */
-void PX4RCOutput::_arm_actuators(bool arm)
```

Comment:
This method and corresponding calls should stay as in https://github.com/ArduPilot/ardupilot/pull/6104